doc/Development_Documentation/21_Deployment/03_Multi_Environment.md

```diff
@@ -33,16 +33,62 @@ var/config/system.php
 > **Note:** If you put your configurations into `app/config/pimcore/` they might not writable by the Pimcore backend UI. 
 > This can be especially useful when having automated building environments and don't want the user to allow changing settings.  
 
-If you add a new environment which is not an existing one by default (those are `dev`, `test` and `prod`), you need to
-manually create a YAML config file for the project. If you set environment before installation, this is needed to be done
-before installation. For instance, for a `staging` environment, you will need to add a `app/config/config_staging.yml` file
-with the following content:
+## Set a new Environment name
 
-```yaml
-imports:
-    - { resource: config.yml }
+You can need to configure a new environment name different than the existing ones, which respect Symfony convention: `dev`, `test` and `prod`.
+
+> **Note:** The default `test` environment should only be used for running test suite (like Travis).
+> It use the session.storage.mock_file which fakes sessions (as consequence, the administrator cannot log in Pimcore for instance)
+
+To configure a new environment, you need to manually create a YAML config file for the project in two places (you can start by copying config_dev.yml for instance in each of those):
+
+```
+app/config/pimcore/
+pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/config/pimcore/
```

Comment:
Changes in the core isn't good idea IMHO. Isn't it possible to do it in some other way?

Reply:
Shouldn't be necessary at all, ```app/config/pimcore``` or ```src/AppBundle/Resources/config``` should be totally enough

Reply:
@Yivan the only necessary file is `app/config/config_<environment>.yml`. Please do not suggest adding/changing core files. IMHO the previous description was fine - I'd just add the note regarding the test environment.